order-dev.js

```diff
@@ -0,0 +1,488 @@
+      }
+      .proof_overlay p {
+        font-size: 0.8em;
+        line-height: 12px;
+        font-weight: bold;
+        white-space: nowrap;
+      }
+      .proof_overlay.top {
+        left: 11px;
+        top: 11px;
+      }
+      .proof_overlay.bottom {
+        right: 11px;
+        bottom: 18px;
+      }
+    </style>
+  `;
+  sortedProofs.forEach((proof) => {
+    result += Render.try('proof', proof);
+  });
+  return result;
+};
+
+Render.proof = (proof) => {
+  var letter = proof.filename.split('_')[1].toLowerCase();
+  var text = (proof.approved) ? `✔ ${letter.toUpperCase()}` : `${letter.toUpperCase()}`;
+  if (proof.date) text += ` • <span class="datetime">${proof.date}</span>`;
+  return `
+    <div class="section" proof="${letter}">
+      <div class="proof_overlay top ${(proof.approved) ? 'approved' : ''}" id="proof_${letter}">
+        <p>${text}</p>
+      </div>
+      <div class="proof_overlay bottom">
+        <p>• click to see full •</p>
+      </div>
+
+      <div class="row">
+
+        <figure>
+          <a href="${proof.url}">
+            <img src="${proof.url}" alt="Proof ${letter.toUpperCase()}"/>
+          </a>
+        </figure>
+      </div>
+    </div>
+  `;
+}
+function ticketItemString(data) {
+  var string = '';
+  string += `${data.custFirst} ${data.custLast},`;
+  string += `${data.email},`;
+  string += `${data.orderId_raw}`;
+
+  return string;
+}
+
+async function createReplyTicket() {
+  var replyText = $("textarea").val();
+  var reply = convertReply($("textarea").val());
+  API.clearCache('v2-getCustomerOrder-dev_' + window.location.search);
+  if(reply && reply.length > 9) {
+    $('div.customerReply').replaceWith(`
+      <div class="customerReply">
+        <div class="section">
+          <div class="replyTitle"><h5>Reply:</h5></div>
+          <div class="spinner"></div>
+          <form id="reply" id="replyForm">
+            <textarea class="replyText" form="replyform" placeholder="Sending reply...\n${replyText}"></textarea>
+          </form>
+        </div>
+      </div>
+    `);
+    console.log("\nCALL v2-custReply: " + new Date());
+    API.call({
+      cacheMS: -1,
+      method: 'v2-customerReplyToFreshdesk',
+      httpMethod: 'POST',
+      body: JSON.stringify({replyText: reply}),
+      onSuccess: (data) => {
+        console.log("\nv2-custReply SUCCESS: " + new Date());
+        console.log("\nDATA stringified = " + JSON.stringify(data));
+
+        if(data.solveRes.records[0].ticket_id) {
+          location.reload();
+        }else{
+          console.log("\nNO solve response");
+        }
+      },
+      onFailure: (data) => {
+        console.log('customer reply fail');
+        $('#main').html(`
+          <div class="row">
+            <p>An error has occured.<br>${data.error}<br>Please try again later.</p>
+          </div>
+        `);
+      }
+    });
+  }else{//no reply text when submit button hit. try again
+    $('div.customerReply').replaceWith(`
```

Comment:
replicated logic.
Why replace the whole form with a fresh one? How about do nothing at all if no text was submitted?